WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailListFragment.kt

```diff
@@ -255,11 +256,14 @@ class NotificationsDetailListFragment : ListFragment(), NotificationFragment {
             }
 
             requireNotNull(notification).let { note ->
-                ReaderActivityLauncher.showReaderComments(
-                    activity, note.siteId.toLong(), note.postId.toLong(),
-                    note.commentId,
-                    COMMENT_NOTIFICATION.sourceDescription
-                )
+                val maybeActivity: FragmentActivity? = activity
+                maybeActivity?.let { fragmentActivity ->
+                    ReaderActivityLauncher.showReaderComments(
+                        fragmentActivity, note.siteId.toLong(), note.postId.toLong(),
+                        note.commentId,
+                        COMMENT_NOTIFICATION.sourceDescription
+                    )
+                }
```

Comment:
```suggestion
                context?.let { nonNullContext ->
                    ReaderActivityLauncher.showReaderComments(
                        nonNullContext, note.siteId.toLong(), note.postId.toLong(),
                        note.commentId,
                        COMMENT_NOTIFICATION.sourceDescription
                    )
                }
```

np: Wouldn't this be simpler?

Reply:
Thanks for the suggestion, I agree that naming is better so I've updated it 👍